runtime/ops/os/mod.rs

```diff
@@ -386,6 +386,39 @@ fn rss() -> usize {
   task_info.resident_size as usize
 }
 
+#[cfg(target_os = "openbsd")]
+fn rss() -> usize {
+  let pid = unsafe { libc::getpid() };
+  let pagesize = unsafe { libc::sysconf(libc::_SC_PAGESIZE) } as usize;
+  let mut kinfoproc: std::mem::MaybeUninit<libc::kinfo_proc> =
+    std::mem::MaybeUninit::uninit();
+  let mut size = std::mem::size_of_val(&kinfoproc) as libc::size_t;
+  let mut mib = [
```

Comment:
**suggestion:** Maybe give pretty gratuitous comments on all the things going on: At least to me all of this, "mib" etc, is just the usual UNIX three letter acronyms with no real meaning. Having some reference point as to what this is doing would help in wanting to keep it in the code base, as opposed to somewhere down the line someone going "oh yeah the OpenBSD support code, yeah no idea what any of that does, no way we can give even the slightest support for it so better to drop it".

Not to suggest that that would happen, not really, but... yeah, comments. I like them.

Reply:
Naming the `name` parameter passed to libc::sysctl is a widely used convention. In deno's codebase, runtime/ops/os/sys_info.rs has several examples that don't expand the abbreviation (Management Information Base) either. Do you think it's particularly neccessary here?

Reply:
Alright, I wasn't familiar with the code around here so ended up spitballing a bad suggestion :) Since it aligns with the codebase it's fine.